doc/api/vm.md

```diff
@@ -91,6 +95,8 @@ changes:
     using it in a production environment.
     * `specifier` {string} specifier passed to `import()`
     * `script` {vm.Script}
+    * `importAssertions` {Object} The `"assert"` value passed to the
+      `optionExpression` optional parameter.
```

Comment:
I don't know what the `optionExpression` is.

Reply:
It doesn't mean much to me either, I'm open to better suggestion. That's how the spec is calling the second (optional) "parameter" of `import()`: https://tc39.es/proposal-import-assertions/#sec-evaluate-import-call

Reply:
In this case, there's a typo. It's `optionsExpression` in the spec.
Can we make it a link to https://tc39.es/proposal-import-assertions/#sec-evaluate-import-call for reference?